Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -411,27 +474,54 @@ public class CarPlayNavigationViewController: UIViewController {
         var buttons: [CPGridButton] = []
         let starImage = UIImage(named: "star", in: .mapboxNavigation, compatibleWith: nil)!
         for i in 1...5 {
-            let button = CPGridButton(titleVariants: ["\(i) star\(i == 1 ? "" : "s")"], image: starImage, handler: buttonHandler)
+            let button = CPGridButton(titleVariants: ["\(i) star\(i == 1 ? "" : "s")"],
```

Comment:
These title variants should be localized, accounting for the fact that pluralization works differently in each language. Here’s an example to follow:

https://github.com/mapbox/mapbox-navigation-ios/blob/39bcdb29a2386bd6d7fd3eb311b7b9bf0a710149/Sources/MapboxNavigation/RatingControl.swift#L104-L109 https://github.com/mapbox/mapbox-navigation-ios/blob/39bcdb29a2386bd6d7fd3eb311b7b9bf0a710149/Sources/MapboxNavigation/Resources/en.lproj/Localizable.strings#L157-L158 https://github.com/mapbox/mapbox-navigation-ios/blob/39bcdb29a2386bd6d7fd3eb311b7b9bf0a710149/Sources/MapboxNavigation/Resources/en.lproj/Localizable.stringsdict#L21-L36